.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+jobs:
+  checkout_cuberite:
+    working_directory: ~/cuberite
+    docker: &cube_docker
+    - image: peterbell10/cuberite-circleci:0.0.3
```

Comment:
I'm not too familiar with Docker, does this mean that it uses something outside the canon repo? Can it be moved into the main repo?

Reply:
Yes, It's on dockerhub and it has to be for Circle to be able to download it.  There is a cuberite dockerhub account though and I think [cuberite/docker-ci](https://github.com/cuberite/docker-ci) might be automatically build by dockerhub.  I'm not sure though, maybe @bearbin knows?

Reply:
Okay, I've had a look at [docker hub's docs](https://docs.docker.com/docker-hub/builds/#understand-the-build-process) and it seems that the dockerfile has to be in its own repo for the automated builds to work.  After that's set up, whoever is in control of the [cuberite dockerhub account](https://hub.docker.com/u/cuberite/) would need to setup an automated build.

Reply:
Alternatively, we could setup a new branch for [cuberite/docker-build-image](https://hub.docker.com/r/cuberite/docker-build-image/) that includes all of the clang stuff.

Reply:
@madmaxoft could you create a new circle-ci branch on the cuberite/docker-build-image github?

Reply:
It's actually literally called [cuberite/docker-build-image](https://github.com/cuberite/docker-build-image).  That's the one that's already setup to build automatically.

Reply:
I've added you there as well.